backend/tardigrade/fs.go

```diff
@@ -219,8 +219,15 @@ func NewFs(name, root string, m configmap.Mapper) (_ fs.Fs, err error) {
 	f.access = access
 
 	f.features = (&fs.Features{
-		BucketBased:       true,
-		BucketBasedRootOK: true,
+		BucketBased:             true,
+		BucketBasedRootOK:       true,
+		CanHaveEmptyDirectories: false,
```

Comment:
This line does not do anything.

Reply:
see https://github.com/storj/rclone/pull/7#discussion_r415979054